website/cue/reference/components/transforms/filter.cue

```diff
@@ -42,11 +43,18 @@ components: transforms: filter: {
 				"""
 			required: true
 			warnings: []
-			type: string: {
+			type: condition: {
+				syntax: ["datadog_search", "vrl_boolean_expression"]
```

Comment:
```suggestion
				type: ["datadog_search", "vrl_boolean_expression"]
```

I believe this is actually called `type` in the config:

https://github.com/vectordotdev/vector/blob/e10294c7b41b7328f708c03a37740c9e617270d8/src/conditions/mod.rs#L31